openapi.yaml

```diff
@@ -723,6 +723,105 @@ paths:
                                 type: array
                                 items:
                                     $ref: '#/components/schemas/Error' 
+                '500':
+                    description: Unexpected error
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    $ref: '#/components/schemas/Error'
+                                example:
+                                    - code: 500
+                                      message: "An unexpected problem occurred"  
+    /projects/{project_id}/assigned_users:
+        get:
+            summary: 'List assigned users'
+            operationId: listProjectAssignedUsers
+            tags:
+                - Projects
+            parameters:
+              - name: project_id
+                in: path
+                description: ID of the Project
+                required: true
+                schema:
+                    type: string
+                    format: id
+              - name: page
+                in: query
+                schema:
+                    type: integer
+                    format: int32
+              - name: per_page
+                in: query
+                schema:
+                    type: integer
+                    format: int32
+            responses:
+                '200':
+                    description: 'List of project assigned users'
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    $ref: '#/components/schemas/User'
+                '400':
```

Comment:
The only circumstance I can see raising this code is if a `Location / User` cannot be found. Is a 400 more preferable to a 404, which is raised by `ActiveRecord::RecordNotFound`?

I could also be overlooking something, in which case, I'd be happy to hear why we want a 400.

Same questions for the `PUT / DELETE` 400 responses too.

Reply:
We should be consistent and 400 - "Not found" is more correct.

Reply:
Sorry, I flipped it 404 is not found while 400 is bad request. I'll fall back to "consistently incorrect" is still desirable. I assume other endpoints are responding with 400 AND modifying those to be 404 is not desirable.

Reply:
To be honest, 400 we're used here just as catch-all 40x's because we haven't mapped out the possible response codes from the API yet. 

I agree 100% that we should use 404 if that's the only 400-level error that could occur?